x-pack/legacy/plugins/infra/public/containers/logs/log_entries/index.ts

```diff
@@ -39,31 +46,33 @@ type ActionObj = ReceiveEntriesAction | FetchOrErrorAction;
 type Dispatch = (action: ActionObj) => void;
 
 interface LogEntriesProps {
+  startDate: string | null;
+  endDate: string | null;
+  startTimestamp: number | null;
+  endTimestamp: number | null;
+  liveStreamingInterval: number;
```

Comment:
The SuperDatePicker specifies how often it should reload. We store it in `LogPositionState` and pass it here.